iroh-net/src/netcheck/reportgen.rs

```diff
@@ -266,11 +266,19 @@ impl Actor {
                 }
 
                 // Drive the probes.
```

Comment:
maybe update this comment to say something like "wait for probe tasks to finish" as this no longer drives the future.